runtime/src/main/scala/akka/grpc/internal/Marshallers.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2018-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.internal
+
+import java.io.{ ByteArrayInputStream, ByteArrayOutputStream, InputStream }
+
+import io.grpc.KnownLength
+import akka.annotation.InternalStableApi
+import akka.grpc.ProtobufSerializer
+
+/**
+ * INTERNAL API
+ */
+@InternalStableApi
+abstract class BaseMarshaller[T](val protobufSerializer: ProtobufSerializer[T])
+    extends io.grpc.MethodDescriptor.Marshaller[T]
+    with WithProtobufSerializer[T] {
+  override def parse(stream: InputStream): T = {
+    val baos = new ByteArrayOutputStream(math.max(64, stream.available()))
+    val buffer = new Array[Byte](32 * 1024)
+
+    // Blocking calls underneath...
+    // we can't avoid it for the moment because we are relying on the Netty's Channel API
```

Comment:
What will be the fix? Do we have an issue for that already? Is that already contained somehow? Probably it only affects the netty backend?

Reply:
Added #1229